### PR TITLE
docs(api): typo

### PR DESCRIPTION
**Pull request by artivilla99**

### CODE REVIEW

**Comment by gr2m** [comment]
good catch :+1: could you maybe reword your commit to `docs(api): typo`? We follow a few conventions: https://github.com/hoodiehq/hoodie/blob/master/CONTRIBUTING.md#commit-message-conventions

You can reword your last commit with `git commit --amend`. Here is some information on it: https://coderwall.com/p/_thg9a/reword-a-git-commit-message

I’m also happy to do it for you, just let me know :)

**Comment by artivilla99** [comment]
Thanks for linking me to that doc. Also, is this for the commit message and the PR description?

**Comment by gr2m** [comment]
Glad to hear it was helpful, thanks for fixing the commit. The conventions are for commit messages only.

Thanks a lot for sending your first PR to Hoodie. Just in case you would like to contribute something else, be it code or documentation or story telling or you name it, feel free to ping me anytime, happy to assist finding something interesting :)
